### PR TITLE
Feature/balancer/pool create liquidity warning

### PR DESCRIPTION
**Pull request by PaulMcInnis**
Liquidity input warning with 5% margin by USD value

image

image

Should go in after #194

### CODE REVIEW

**Comment by PaulMcInnis** [comment]
OK - going to split off the logic for this verification into its own hook

File: packages/berajs/src/hooks/modules/dex/useLiquidityMismatch.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+      return;
+    }
+
+    setLiquidityMismatchInfo({
+      title: `You could lose up to $${totalMismatchedLiquidityUSD.toFixed(
+        2,
+      )} (${(totalMismatchPercentage * 100).toFixed(2)}%)`,
+      message: `Based on current token USD pricing in terms of HONEY, the value of tokens added does not align with the 
+      specified pool weights. This discrepancy could expose you to potential losses from arbitrageurs.`,
+    });
```
Comment (bearpong) [inline packages/berajs/src/hooks/modules/dex/useLiquidityMismatch.ts:122]: This refers to HONEY but it depends on the input tokens. Probably we might just use a more generic message:

`Based on the market token prices, the value of tokens does not align with the specified pool weights. This discrepancy could expose you to potential losses from arbitrageurs.`

**Review by bearpong — CHANGES_REQUESTED** [review]
Made a smol request for the warning copy. What happens if token are not listed on BEX and we can't determine their USD values? maybe we should show an alternative warning that just tells user to pay attention

**Comment by PaulMcInnis** [comment]
OK going to add a default warning for if there is no price information available 👍

**Review by bearpong — APPROVED** [review]
lesgo

**Pull request merged**